package.json

```diff
@@ -89,6 +89,7 @@
     "eslint-plugin-react": "^7.9.1",
     "fs-extra": "^7.0.0",
     "grommet-icons": "^3.1.0-beta.1",
+    "grommet-theme-hpe": "^0.4.0",
```

Comment:
Can we remove this? It doesn't seem right to depend on an external theme?

Reply:
we are using it for testing and storybook. it is a devDependency, so the published grommet is not dependent on that.